Gems/ROS2PoseControl/Code/Source/Clients/ROS2PoseControl.cpp

```diff
@@ -269,6 +289,11 @@ namespace ROS2PoseControl
 
     void ROS2PoseControl::ApplyTransform(const AZ::Transform& transform)
     {
+        if (!m_isPhysicsDisabled)
+        {
+            DisablePhysics();
```

Comment:
Well yes and no, the problem is that this function can be called before first tick, and must be called after whole tree of entities activates, therefore I thought that adding it here would be the best option. It's one check of an boolean and this way we can be 100% sure that model won't be split in half because movement happened before deactivation.